pygeoapi/templates/item.html

```diff
@@ -37,7 +37,28 @@ <h2>Item {{ data['id'] }}</h2>
       </div>
       <div class="row">
         <div class="col-md-6 col-sm-12">
-          <div id="items-map"></div>
+          <div class="row">
+            <div class="col-sm-12">
+              <div id="items-map"></div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-sm-12">
+              {% if data['previous'] or data['next'] %}
+              <div class="row">
+                <div class="col-sm-12">
+                  {% for link in data['links'] %}
+                  {% if link['rel'] == 'prev' %}
+                  <a role="button" href="./{{ data['previous'] }}">Prev</a>
```

Comment:
@tomkralidis Please ensure these lines have "prev" instead of "previous" to not break things. Otherwise all good.